docs/book/src/installation/host/configuration.rst

```diff
@@ -89,6 +89,9 @@ assume you're going to use KVM. Please see the latest version here: `kvm.conf`_.
 
 .. _`kvm.conf`: https://github.com/kevoreilly/CAPEv2/blob/master/conf/kvm.conf
 
+If you are using KVM (kvm.conf), for each VM you want to use for analysis there must be a dedicated section. First you have to create and configure the VM (following the instructions in the dedicated chapter, see :ref:`preparing_the_guest`). The name of the section must be the same as the label of the VM as printed by ``$ virsh list --all``. If no VMs are shown, you can execute the following command sequence: ``$ virsh``, ``$ connect qemu:///system``, ``$ list --all``; or you can check `this link <https://serverfault.com/a/861853>`_ to learn how to change the connection in Virtual Manager.
```

Comment:
FYI im adding that to bash/zsh https://github.com/doomedraven/Tools/blob/master/Virtualization/kvm-qemu.sh#L260-L264